css/style.css

```diff
@@ -0,0 +1,342 @@
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+.top {
+  width: 100%;
+  max-width: 640px;
+  text-align: center;
+  margin: auto;
+  background-image: url(../imgs/mobile-bg.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.welcome-content {
+  padding: 200px 20px 75px 20px;
+  text-align: left;
+}
+
+.welcome-content-h2 {
+  font-weight: 400;
+  font-size: 20px;
+  color: #ff4f38;
+}
+
+.welcome-content-h1 {
+  font-weight: 900;
+  font-size: 30px;
+  background-image: url(../imgs/backgrounds/welcome-prism-bg.png);
+  background-clip: inherit;
+  -webkit-background-clip: text;
+  color: transparent;
+  background-repeat: no-repeat;
+  background-position: center center;
+  margin-bottom: 100px;
+}
+
+.paragraph {
+  margin: 22px 0 22px 0;
+  padding: 16px;
+  background-color: #f7f7f7;
+  border: 2px solid white;
+}
+
+.paragraph p {
+  font-weight: 500;
+  line-height: 27px;
+  font-size: 12px;
+}
+
+.welcome-content > h3 {
+  font-weight: 800;
+  margin-top: 14px;
+  margin-bottom: 10px;
+}
+
+.welcome-content > h4 {
+  font-size: 14px;
+}
+
+span {
+  display: block;
+}
+
+.main-programme {
+  background-image: url(../imgs/backgrounds/crosspattern.svg);
+  max-width: 640px;
+  width: 100%;
+  text-align: center;
+  margin: auto;
+  background-position: center;
+}
+
+.main-programme-content h3 {
+  color: #f7f7f7;
+  padding-top: 32px;
+  padding-bottom: 15px;
+}
+
+.line1 {
+  border: 1px solid #ec5242;
+  width: 35px;
+  margin: auto;
+  margin-bottom: 25px;
+}
+
+.line {
+  border: 1px solid #d3d3d3;
+  width: 35px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.programme-cards {
+  display: flex;
+  flex-direction: column;
+}
+
+.programme-card {
+  display: grid;
+  grid-template-columns: 1.5fr 2.5fr 6fr;
+  background-color: rgba(255, 255, 255, 0.1);
+  margin: 8px;
+  padding: 13px;
+  align-items: center;
+}
+
+.programme-card > * {
+  width: 100%;
+}
+
+.programme-card > h4 {
+  padding-right: 10px;
+  color: #ec5242;
+}
+
+.programme-card > p {
+  color: #f7f7f7;
+  text-align: left;
+}
+
+.main-button {
+  margin: 25px auto 25px auto;
+  background-color: #ff4f38;
+  border: none;
+  color: #f7f7f7;
+  font-size: 16px;
+  font-weight: bolder;
+  font-weight: 400;
+  padding: 24px 40px;
+}
+
+.see-programme {
+  display: none;
+}
+
+/* Speakers Section */
+
+.featured-speakers {
+  display: grid;
+  text-align: center;
+  width: 100%;
+  max-width: 640px;
+  margin: auto;
+}
+
+.speaker-contents {
+  text-align: left;
+}
+
+.speaker-cards {
+  margin: 8px;
+}
+
+.speaker-contents > h3 {
+  padding-top: 32px;
+  padding-bottom: 15px;
+  text-align: center;
+}
+
+.speaker-card {
+  display: grid;
+  width: 100%;
+  max-width: 640px;
+  grid-template-columns: 1fr 3.5fr;
+  margin: auto;
+}
+
+.speaker-img {
+  align-self: center;
+}
+
+.speaker-img > img {
+  width: 100%;
+}
+
```

Comment:
- [ ] Kindly add a checked background on the `speaker-card` before the image as seen in the original design.

![image](https://user-images.githubusercontent.com/4247740/164878151-a34795aa-db02-433e-a8a2-10e3cb433d9c.png)

Reply:
@Cathella  thanks for the code review, I fixed the problems now.